virtcontainers/fc.go

```diff
@@ -682,25 +701,11 @@ func (fc *firecracker) createDiskPool() error {
 		isReadOnly := false
 		isRootDevice := false
 
-		// Create a temporary file as a placeholder backend for the drive
-		//hostURL, err := fc.store.Raw("")
-		hostURL, err := fc.store.Raw(driveID)
+		jailedDrive, err := fc.createJailedResource(driveID)
```

Comment:
I think the comment is useful and valid with the change, can we keep it ?

Reply:
ok sounds good